src/components/Cards/LegislationCard/LegislationCard.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+
+import LegislationAdoptedIcon from "./LegislationAdoptedIcon";
+import LegislationRejectedIcon from "./LegislationRejectedIcon";
+import LegislationInProgressIcon from "./LegislationInProgressIcon";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+const StatusIconContainer = styled.div({
+  float: "right",
+  marginRight: 8,
+  height: 24,
+  width: 24,
+});
+
+export interface LegislationCardProps {
+  /**The name of the legislation */
+  name: string;
+  /**The status of legislation. If legislation is in progress then the status is Next Meeting */
+  status: string;
+  /**The legislation status's date or the date of the next meeting */
+  date?: string;
+  /** List of legislation keywords */
+  tags: string[];
+}
+
+const LegislationCard: FC<LegislationCardProps> = ({
+  name,
+  status,
+  date,
+  tags,
+}: LegislationCardProps) => {
+  const tagString = tags.join(` • `);
+
+  let statusIcon;
+
+  if (status === "Adopted") {
+    statusIcon = <LegislationAdoptedIcon />;
+  } else if (status === "Rejected") {
+    statusIcon = <LegislationRejectedIcon />;
+  } else {
+    statusIcon = <LegislationInProgressIcon />;
+  }
+
+  return (
+    <section className="mzp-c-card mzp-has-aspect-16-9">
+      <div className="mzp-c-card-block-link">
```

Comment:
What was the reason that this card-block-link element was turned into a div and is no longer a link element?

Seems to deviate from Protocol documentation examples:
https://protocol.mozilla.org/patterns/molecules/card.html

Reply:
I just read through the comments you gave me in my PR and realized you already answered this exact question!

Keeping card routing unopinionated sounds perfectly reasonable to me.

Going to resolve this conversation and feel free to merge whenever!